framework/math/quadratures/angular/product_quadrature.cc

```diff
@@ -145,7 +143,7 @@ ProductQuadrature::OptimizeForPolarSymmetry(const double normalization)
 
 AngularQuadratureProdGL::AngularQuadratureProdGL(int Nphemi, bool verbose) : ProductQuadrature()
```

Comment:
what is the purpose of a product Gauss Legendre quad? The only one that matters is a Gauss Legendre Chebyshev.